<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="grpbx_browser">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="title">
      <string>Браузер</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lbl_scale">
          <property name="font">
           <font>
            <family>Arial</family>
           </font>
          </property>
          <property name="text">
           <string>Масштаб:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spnbx_scale">
          <property name="font">
           <font>
            <family>Arial</family>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::ClickFocus</enum>
          </property>
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="minimum">
           <number>25</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpbx_keys">
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="title">
      <string>Комбинации клавиш</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lbl_prevquestion">
          <property name="font">
           <font>
            <family>Arial</family>
           </font>
          </property>
          <property name="text">
           <string>К пред. заданию</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pshbttn_prevquestion">
          <property name="font">
           <font>
            <family>Arial</family>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::ClickFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #FFFFFF</string>
          </property>
          <property name="text">
           <string>Клавиша</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="lbl_nextquestion">
          <property name="font">
           <font>
            <family>Arial</family>
           </font>
          </property>
          <property name="text">
           <string>К след. заданию</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pshbttn_nextquestion">
          <property name="font">
           <font>
            <family>Arial</family>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::ClickFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #FFFFFF</string>
          </property>
          <property name="text">
           <string>Клавиша</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="lbl_confirm">
          <property name="font">
           <font>
            <family>Arial</family>
           </font>
          </property>
          <property name="text">
           <string>Подтвердить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pshbttn_confirm">
          <property name="font">
           <font>
            <family>Arial</family>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::ClickFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #FFFFFF</string>
          </property>
          <property name="text">
           <string>Клавиша</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="lbl_refresh">
          <property name="font">
           <font>
            <family>Arial</family>
           </font>
          </property>
          <property name="text">
           <string>Обновить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pshbttn_refresh">
          <property name="font">
           <font>
            <family>Arial</family>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::ClickFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #FFFFFF</string>
          </property>
          <property name="text">
           <string>Клавиша</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpbox_about">
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="title">
      <string>О программе</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pshbttn_authorpage">
        <property name="font">
         <font>
          <family>Arial</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #FFFFFF</string>
        </property>
        <property name="text">
         <string>Репозиторий на GitHub</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pshbttn_bank">
        <property name="font">
         <font>
          <family>Arial</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #FFFFFF</string>
        </property>
        <property name="text">
         <string>Сайт ФИПИ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_version">
        <property name="font">
         <font>
          <family>Arial</family>
         </font>
        </property>
        <property name="text">
         <string>Версия 0.2.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
